src/main/scala/faces/apps/ModelViewer.scala

```diff
@@ -382,7 +382,7 @@ case class SimpleModelViewer(
     if (fc.showSaveDialog(null) == JFileChooser.APPROVE_OPTION) {
       var file = fc.getSelectedFile
       if (file.isDirectory) file = new File(file,"instance.png")
-      if ( !file.getName.endsWith(".png")) file = new File( file+".png")
+      if ( !file.getName.endsWith(".png")) file = new File( file.getAbsolutePath()+".png")
```

Comment:
Could you please remove the empty parenthesis? This will fix a warning in the IDE (intellij).